contracts/strategies/strategy-base.sol

```diff
@@ -22,6 +22,7 @@ abstract contract StrategyBase {
 
     // Dex
     address public pangolinRouter = 0xE54Ca86531e17Ef3616d22Ca28b0D458b6C89106;
```

Comment:
might make sense to rename this to simply "router" or "dexRouter" if there is a question about not always using Pangolin

Reply:
yeah, was looking at refactoring much of this for better flexibility, but for now, just get it stood up!